### DISCUSSION TITLE
various entities captured by lelegofrog

### DISCUSSION THREAD

**Discussion by city41** [city41/smaghetti#149] [General]
![image](https://user-images.githubusercontent.com/141159/138777229-4dc7d11d-81e9-441f-8f94-4be52345f593.png)

**Comment by lelegofrog** [comment]
some more underground grass object tiles
![image](https://user-images.githubusercontent.com/61850335/138777818-490eb7ec-1c23-44bf-bd27-966e32cb1b1b.png)

**Reply by lelegofrog** [reply]
![image](https://user-images.githubusercontent.com/61850335/138784842-1d66ea7d-cad2-45a9-8e95-a8ad443498f8.png)
you want more? i'll show you more

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/138778237-6eca947d-b586-4543-a04e-f5dec4d06939.png)
more things!!!

**Reply by lelegofrog** [reply]
i think that supposed "goal tape" piece is a part of a cannon thing, but i don't remember what

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/138949700-eb0be538-8555-46d8-8239-c2350a1c478f.png)
i enjoy the more

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/139180290-d0df522d-ad14-4662-a060-d9b67214e3e9.png)
![image](https://user-images.githubusercontent.com/61850335/139180303-037df195-c3b4-4328-84f1-047856c0c7b1.png)
smallest

**Reply by city41** [reply]
this is the brick pyramid. It can get up to four rows tall, but it easily gets cut off. In this case, I believe the wood block cut it off. It's in smaghetti, you can play around with it.

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/139183376-824871c6-6aa1-4995-a0d5-59b5ccb31fed.png)
more things!!!!!

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/139183739-6bda5c42-62ad-449b-8491-eead2c7bf563.png)

**Reply by lelegofrog** [reply]
![Screenshot 2022-01-27 10 27 22](https://user-images.githubusercontent.com/61850335/151411809-69aa08ff-36b1-4dc5-aead-63c138462193.png)
![Screenshot 2022-01-27 10 27 29](https://user-images.githubusercontent.com/61850335/151411813-3a7a0164-2bbd-424c-9a43-63d3850aa6ef.png)
![Screenshot 2022-01-27 10 26 12](https://user-images.githubusercontent.com/61850335/151411810-f6044599-4d07-44a5-886f-d25de6e93fd7.png)
![Screenshot 2022-01-27 10 26 19](https://user-images.githubusercontent.com/61850335/151411812-27fb5887-144a-492e-b69d-f4c85c6e0704.png)
object set 1

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/152428635-9587af29-461b-4045-bced-8d6fe1f3bc42.png)

**Reply by lelegofrog** [reply]
there are also a lot more airship level chunks around where i found the one with the bill blaster
